src/systems/userinput/devices/webxr-controller.js

```diff
@@ -0,0 +1,75 @@
+import { paths } from "../paths";
+import { Pose } from "../pose";
+
+const ONES = new THREE.Vector3(1, 1, 1);
+const LEFT_HAND_OFFSET = new THREE.Matrix4().makeTranslation(-0.025, -0.03, 0.12);
```

Comment:
Noting this for later -- these offsets should probably depend on the specific controller being used.

Reply:
Ah, good point. I'll add a comment to the code.